notebooks/118-optimize-preprocessing/118-optimize-preprocessing.ipynb

```diff
@@ -14,7 +14,6 @@
     "This tutorial include following steps:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    print("Result of inference with Preprocessing API:")
</pre><p>Cell output includes previous logs with Model Optimizer:</p><pre class="ql-syntax" spellcheck="false">Result of inference for preprocessing with ModelOptimizer:
n02099601 golden retriever, 0.56439
n02098413 Lhasa, Lhasa apso, 0.35731
n02108915 French bulldog, 0.00730
n02111129 Leonberg, 0.00687
n04404412 television, television system, 0.00317
</pre><p>Maybe we should delete it from cell output or re-execute the cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Everything else LGTM

Reply:
updated cell output